### ISSUE TITLE
edit tunnel ?

### ISSUE THREAD

**Issue by mickaelmonsieur** [jdrew82/netbox-tunnels-plugin#3]
Hello,
Do you plan to make an edit tunnel button?

**Comment by jdrew82** [comment]
Yes, this plugin is intended to be fully featured eventually. The current focus is on the Nautobot version.
